### DISCUSSION TITLE
使用感受

### DISCUSSION THREAD

**Discussion by GaoNeng-wWw** [DevCloudFE/vue-devui#1422] [Show and tell]
独自一人使用组件发现的一些不知道算不算坑的坑。诚邀各位组件开发者一起来讨论

## 使用组件

- button
- button-group
- layout
- grid
- icon
- collapse
- Menu

## button

|页面名|坑|解决方案
|:-:|:-:|:-:|
|/|不结合button-group会出现高度问题|加了一个.btn的class，手动设置32px的高度

@Zcating @daviForevel @vaebe @Lonely-shang @qinwencheng 

## button-group

|页面名|坑|解决方案
|:-:|:-:|:-:|
|/|没办法定义两个button间的间隙|不用button-group

@Zcating @daviForevel @vaebe @Lonely-shang @qinwencheng 

## grid

|页面名|坑|解决方案
|:-:|:-:|:-:|
|*|flex状态下wrap方向无法修改|设置一个原子化的`flex-wrap-reverse`的class，

@ming-bin @liyao1520 @Lonely-shang 

## Menu

|页面名|坑|解决方案
|:-:|:-:|:-:|
|/doc/*|Menu菜单按下后整个页面会刷新(不排除是我路由设置有问题)|自己维护了一个Menu组件
|/doc/*|二级或多级菜单下Menu-Item的icon与text相距过远问题|自己维护了一个Menu组件

@GaoNeng-wWw

**Comment by vaebe** [comment]
button-group 这个设计上 不太支持间隙，如何展示的问题解决了 就可以考虑加

**Reply by GaoNeng-wWw** [reply]
或许可以用flex，然后使用gap属性？🤔
